scene/gui/rich_text_label.cpp

```diff
@@ -1197,6 +1197,46 @@ int RichTextLabel::_draw_line(ItemFrame *p_frame, int p_line, const Vector2 &p_o
 					Transform2D char_xform;
 					char_xform.set_origin(p_ofs + off_step);
 
+					bool is_in_transition = false;
+					if (visibility_transition.is_valid()) {
+						int char_key = processed_glyphs_step;
+						print_line(vformat("GLY %s", processed_glyphs_step));
```

Comment:
Remove debugging print (this can cause large slowdowns, so I wouldn't even print it in verbose mode):

```suggestion
```